velox/functions/lib/ArraySort.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/expression/VectorFunction.h"
+#include "velox/functions/lib/SimpleComparisonMatcher.h"
+
+namespace facebook::velox::functions {
+
+/**
+ * @param ascending If true, sort in ascending order; otherwise, sort in
+ * descending order.
+ * @param nullsFirst If true, nulls are placed first; otherwise, nulls are
+ * placed last.
+ * @param throwOnNestedNull If true, throw an exception if a nested null is
+ * encountered.
+ */
```

Comment:
nit: use `///` for multiple-line comments of a public API. Ditto for the others.